docs/source/guide/scim_workflow.md

```diff
@@ -0,0 +1,112 @@
+---
+title: How SCIM works with Label Studio Enterprise
+short: SCIM workflows and API
+tier: enterprise
+type: guide
+order: 0
+order_enterprise: 492
+meta_title: How SCIM works with Label Studio Enterprise
+meta_description: A description of SCIM workflows and entities
+section: "Install"
+parent: "scim_setup"
+parent_enterprise: "scim_setup"
+date: 2024-01-25 10:07:22
+---
+
+## SCIM workflow
+
+You can use SCIM to do the following:
+
+* Add users
+* Remove users (set their user role to Deactivated)
+* Assign users to groups
+* Unassign a user from a group
+* Map groups to user roles
+    
+    Note that groups are defined in the IdP, not Label Studio. However, the group to role mapping is defined in Label Studio. 
+
+## SCIM API endpoint
+
+You can use SCIM with Label Studio to control and interact with two entities: Users and Groups. 
+
+!!! note
+    Our API uses the [django-scim2 library](https://django-scim2.readthedocs.io/en/latest/). 
+
+#### Users
+* Search for users: `GET /scim/v2/Users?filter=userName =<user@email.com>&startIndex=1&count=100`
+    * `200` response if user exists
+    * `404` if not 
+* Get user: `GET /scim/v2/Users/user@email.com`
+* Create user: `POST /scim/v2/Users/` 
+
+    This will also require a payload that includes user information such as email and password. 
+
+**SCIM user parameters:**
```

Comment:
not sure about these parameters are correct.. let's remove this section with parameters until somebody asks us.

Reply:
Removed